draft-ietf-lamps-csr-attestation.md

```diff
@@ -819,345 +375,62 @@ The current registry values can be retrieved from the IANA online website.
-{{I-D.ietf-rats-msg-wrap}}.
+## Relationship of Attestations and Certificate Extensions
 
-The encoding of this KAT-PAT bundle is shown in the example below.
+Attestations are intended as additional information in the issuance process, and may include sensitive information about the platform, such as hardware details or patch levels, or device ownership. It is NOT RECOMMENDED for a CA to copy attestations into the published certificate. CAs that choose to republish attestations in certificates SHOULD review the contents and delete any sensitive information.
 
-~~~
-EvidenceBundle
-   +
-   |
-   + Evidences
-   |
-   +---->  EvidenceStatement
-        +
-        |
-        +-> type: OID for CMW Collection
-        |         1 3 6 1 5 5 7 1 TBD
-        |
-        +-> stmt: KAT/PAT CMW Collection
-~~~
-
-The value in EvidenceStatement->stmt is based on the
-KAT/PAT example from {{Section 6 of I-D.bft-rats-kat}} and
-the result of CBOR encoding the CMW collection shown below
-(with line-breaks added for readability purposes):
-
-~~~
-{
-  "kat":
-    h'd28443A10126A058C0A30A5820B91B03129222973C214E42BF31D68
-      72A3EF2DBDDA401FBD1F725D48D6BF9C8171909C4A40102200121
-      5820F0FFFA7BA35E76E44CA1F5446D327C8382A5A40E5F29745DF
-      948346C7C88A5D32258207CB4C4873CBB6F097562F61D5280768C
-      D2CFE35FBA97E997280DBAAAE3AF92FE08A101A40102200121582
-      0D7CC072DE2205BDC1537A543D53C60A6ACB62ECCD890C7FA27C9
-      E354089BBE13225820F95E1D4B851A2CC80FFF87D8E23F22AFB72
-      5D535E515D020731E79A3B4E47120584056F50D131FA83979AE06
-      4E76E70DC75C070B6D991AEC08ADF9F41CAB7F1B7E2C47F67DACA
-      8BB49E3119B7BAE77AEC6C89162713E0CC6D0E7327831E67F3284
-      1A',
-  "pat":
-    h'd28443A10126A05824A10A58205CA3750DAF829C30C20797EDDB794
-      9B1FD028C5408F2DD8650AD732327E3FB645840F9F41CAB7F1B7E
-      2C47F67DACA8BB49E3119B7BAE77AEC6C89162713E0CC6D0E7327
-      831E67F32841A56F50D131FA83979AE064E76E70DC75C070B6D99
-      1AEC08AD'
-}
-~~~
-
-## Confidential Compute Architecture (CCA) Platform Token in CSR
+## Additional Security Considerations
 
-The Confidential Compute Architecture (CCA) Platform Token is described in
-{{I-D.ffm-rats-cca-token}} and is also based on the EAT format.  Although the
-full CCA attestation is composed of Realm and Platform Evidence, for the purposes
-of this example only the Platform token is provided.
+In addition to the security considerations listed here, implementers should be familiar with the security considerations of the specifications on which this specification depends: PKCS#10 {{RFC2986}}, CRMF {{RFC4211}}, as well as general security concepts relating to remote attestation; many of these concepts are discussed in {{Section 6 of RFC9334}}, {{Section 7 of RFC9334}}, {{Section 9 of RFC9334}}, {{Section 11 of RFC9334}}, and {{Section 12 of RFC9334}}. Implementers should also be aware of any security considerations relating to the specific attestation formats being carried within the CSR.
+ 
```

Comment:
Should a comment be made here with respect to Certification Policy Statements and the inclusion of appropriate guidance for certificate requesters?